client/src/components/UserAvatar/styles.css

```diff
@@ -1,19 +1,16 @@
+.fit-parent {
+    width: 100%;
+    height: 100%;
+}
 .user-avatar-container .user-avatar-spinner {
-    line-height: 100px;
-    margin-top: 30px;
+    line-height: 200px;
     text-align: center;
     vertical-align: middle;
 }
-
-.user-avatar-container .avatar-picture {
-    text-align: center;
-    margin-top: 30px;
-    margin-bottom: 20px;
-}
-
 .user-avatar-container .avatar-picture img {
-    width: 100%;
     max-width: 200px;
+    max-height: 200px;
```

Comment:
Check this example https://codepen.io/anon/pen/PydRLE
Try add **max-width/width** to image wrapper.
It is not mistake, but can help to manage layout on small screens. You just change properties for only one element.